recipe/build-core.bat

```diff
@@ -1,5 +1,6 @@
 cd python
 set SKIP_THIRDPARTY_INSTALL=1
+set BAZEL_SH="C:\Program Files\Git\bin\bash.exe"
```

Comment:
This should be pointing to the m2-bash from conda-forge. I tested this locally, and I believe the path should be:
```suggestion
set "BAZEL_SH=%PREFIX%\Library\usr\bin\bash.exe"
```
Also note that batch-files need the quotes before the variable (yeah... windows)